release-notes/v1_68.md

```diff
@@ -296,6 +296,30 @@ The CLI is available in a new [devcontainers/cli](https://github.com/devcontaine
 
 The CLI is under active development and will continue evolving to better support more scenarios, such as greater support for individual users. We'd love to hear your feedback along the way, so we've [opened an issue](https://github.com/devcontainers/cli/issues/7) specifically for feedback on the CLI and welcome additional issues and PRs in the repo.
 
+### Dev Container in CI
+
+A GitHub Action and an Azure DevOps Task are available for running a repository's dev container in continuous integration builds. This allows you to reuse the same setup you are using locally for development to build and test your code in CI. See the [documentation](https://github.com/devcontainers/ci/blob/main/README.md) for more details.
```

Comment:
It might be nice to include the acronym the first time.

```suggestion
A GitHub Action and an Azure DevOps Task are available for running a repository's dev container in continuous integration (CI) builds. This allows you to reuse the same setup you are using locally for development to build and test your code in CI. See the [documentation](https://github.com/devcontainers/ci/blob/main/README.md) for more details.
```